src/releaser.ts

```
@@ -229,10 +217,8 @@ class Releaser {
 
         if (!res.id) {
           // SOMETHING when wrong
-          error(
-            'Something went wrong while upload release assets: ' + res.message
-          );
-          return {};
+          error('Something went wrong while upload release assets: ' + res.message);
+          process.exit(1);
```

@yoramdelangen, this is the main line. Making sure the workflow throws an error instead of continueing